python/tests/gold/acceptancegrpcbridgetest/snapshots/econf.json

```diff
@@ -6,7 +6,7 @@
                 "dns_lookup_family": "V4_ONLY",
                 "lb_policy": "ROUND_ROBIN",
                 "load_assignment": {
-                    "cluster_name": "cluster_127_0_0_1_8877",
+                    "cluster_name": "cluster_127_0_0_1_8877_default",
```

Comment:
The `cluster_127_0_0_1_8877` gets renamed to `cluster_127_0_0_1_8877_default` everywhere! I hope that's not an issue 🤞